Framework/DataHandling/inc/MantidDataHandling/SaveStl.h

```diff
@@ -18,6 +18,12 @@ class MeshObject;
 namespace DataHandling {
 enum class ScaleUnits;
 class DLLExport SaveStl {
+  /**
```

Comment:
For doxygen to attach this to the correct class it need to go above the defintion:
```c++
   * Class to contain functionality for writing out STL files for
   * SaveShapeAndEnvironment. handles actual writing to file, creating the
   * header, and removing the scale applied when loading.
   *
   */
class DLLExport SaveStl {
```